### ISSUE TITLE
Error in fixture.beforeEach hook - Unhandled promise rejection [object Object] - not sure what it means

### ISSUE THREAD

**Issue by bbonczek** [DevExpress/testcafe#2931]
### Are you requesting a feature or reporting a bug?
A feature

### What is the current behavior?
Tests fail with this error:

```
1) - Error in fixture.beforeEach hook - 
Unhandled promise rejection: 
[object Object] 
Browser: HeadlessChrome 69.0.3497 / Windows 10.0.0
```
This error message does not really tell what is wrong.

### What is the expected behavior?
Error message should be more descriptive.

### How would you reproduce the current behavior (if this is a bug)?
Not really sure - it is randomly if we run around ~200 tests on 6 chromes in parallel.

Sample beforeEach section for one of our tests that is failing:

`fixture 'xxx'
      .page(config.baseUrl)
      .beforeEach(async t => {
          await t
              .click(traceSelectionPage.openTraceConfigurationButton)
              .click(traceSelectionPage.contentListElement.nth(0))
              .click(traceSelectionPage.contentListElement.nth(0))
              .click(traceSelectionPage.contentListElement.withText('File 1'))
              .click(traceSelectionPage.contentListElement.nth(0))
              // select 2 parameters
              .click(traceSelectionPage.contentListElement.nth(0))
              .click(traceSelectionPage.contentListElement.nth(1))
              .click(traceSelectionPage.addParameterButton)
              .click(propertiesPage.okPropertiesButton);
      });`

### Specify your

* operating system:
Windows 10
* testcafe version:
0.22.0
* node.js version:
v8.12.0

Tests are runned on Google Chrome Version 69.

**Comment by AlexKamaev** [comment]
There is some unhandled promise rejection occurred in our server code or yours. Could you please install latest testcafe@alpha version via `npm i testcafe@alpha` command? If the problem still occurrs, there will be a detailed error message. Could you send it to me?

**Comment by bbonczek** [comment]
Actually nvm, I runned it with testcafe 0.22.0 again - I'll provide another log of testcafe in alpha version soon - it takes some time to run those tests ;)

**Comment by bbonczek** [comment]
Hello, sorry for the late response.
I have now reliable method of reproducing this issue so below are my results for different versions of testcafe:

0.22.0 -> The issue is happening
0.22.0 with -u flag -> The issue is not happening
0.22.1-alpha.3 -> Tests just stop at some point and won't proceed.

The command I've used is:
```
testcafe chrome e2e/ -c 6    // (in one case with additional -u flag)
```
Seems like this issue is pretty similiar to #2852 and #2917 .

**Comment by AlexKamaev** [comment]
As I see from your logs you have another issue with the `0.22.1-alpha.3` version. Does the `unhandled promise rejection` issue appear in this version? In any case it's difficult to determine the cause of the issue without a working sample. Could you please provide me with test files and a sample project or a link to the web page?

**Comment by bbonczek** [comment]
Unfortunatelly I can't share my code, but I tried it again today, and it works now - not sure what happened yesterday. Could you estimate when are you planning to release non alpha version with 66b8c31 fix ?

**Comment by AlexKamaev** [comment]
We are going to release the new TestCafe version this week if everything goes as planned

**Comment by bbonczek** [comment]
Ok great :) Closing this issue then.
